### PR TITLE
Add support for nesting tables via NestedTableCol type

### PR DESCRIPTION
**Pull request by jonathanunderwood**
As discussed in #27 the ability to nest tables within tables is a very useful and common feature. This PR attempts to provide such functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -199,6 +202,26 @@ added into the form.]]
...
+
+Eg:
+
+```python
+class MySubTable(Table):
+    a = Col('1st nested table col')
+    b = Col('2nd nested table col')
+
+class MainTable(Table):
+    id = Col('id)
```
Comment (plumdog) [inline README.md]: Missing closing ' for the string.

File: flask_table/columns.py [hunk 2]
```diff
@@ -252,3 +252,31 @@ def td_contents(self, item, attr_list):
...
+    defined for the nested table.
+
+    Eg:
+
+    class MySubTable(Table):
+        a = Col('1st nested table col')
+        b = Col('2nd nested table col')
+
+    class MainTable(Table):
+        id = Col('id)
```
Comment (plumdog) [inline flask_table/columns.py]: Same here.

**Comment by plumdog** [comment]
Looks :+1: to me, except for a pair of typos.
Lovely work.

**Comment by jonathanunderwood** [comment]
Thanks for looking it over - I have fixed up those two typos by updating the commits, so we should be good to go!

**Comment by plumdog** [comment]
Included in [`0.2.12`](https://github.com/plumdog/flask_table/releases/tag/v0.2.12) and [released to PyPI](https://pypi.python.org/pypi/Flask-Table/0.2.12).

**Comment by jonathanunderwood** [comment]
Thanks!

**Pull request merged**